packages/world/src/VoxelCoord.sol

```diff
@@ -96,11 +97,29 @@ library VoxelCoordLib {
   }
 
   function toForceFieldShardCoord(VoxelCoord memory coord) internal pure returns (VoxelCoord memory) {
-    return toShardCoord(coord, FORCE_FIELD_SHARD_DIM, true);
+    return toShardCoord(coord, FORCE_FIELD_SHARD_DIM, false);
```

Comment:
Force fields are 3D so we dont want to ignore Y here, but energy pools are 2D so we want to ignore Y there